mkdocs.yml

```diff
@@ -18,6 +18,7 @@ nav:
         - Creating a Network From Scratch: Getting Started/Creating-A-Network-From-Scratch.md
         - Running Quorum: Getting Started/running.md
         - Quorum API: Getting Started/api.md
+        - CLI arguments: Getting Started/command-line-arguments.md
```

Comment:
I think we don't really need this page as it seems to be a duplicated of `geth --help`

Reply:
I can expand it a bit more as I agree that they are quite similar. Ideally `geth --help` should be a one liner of the flag, whereas this page can give a bit more description.
The flags will likely be explained further (or repeated) in their respective doc pages, but it is helpful IMO to have an aggregate page of all flags.

Reply:
CLI flags are expected to be explained well in their respected features doc. Creating a central page  is useful but I see some maintenance overhead and duplication of docs here.

Reply:
We can add it later with a more comprehensive set of options.